docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -174,35 +174,38 @@ See linked documentation for more information :ref:`Treatment and management for
     - [Isanaka_etal_2021]_ , [Zw_2020]_
     - This is for SAM-OTP which is ~98% of SAM.
   * - SAM alternative
-    - 90%
+    - 70%
     - 75%
-    - 0.9 x 0.75 = 0.675
-    - 1 - 0.675 = 0.325
+    - 0.7 x 0.75 = 0.53
+    - 1 - 0.53 = 0.47
     - Sphere standards
     - Sphere guideline for efficacy only
   * - MAM baseline
-    - 48.8% (95% CI: 37.4, 60.4)
+    - 15% (95% CI: 10, 20)
     - 73.1% (95% CI:58.5-87.7) for RUSF
-    - 0.488 x 0.731 = 0.34
-    - 1 - 0.34 = 0.66
-    - [Ackatia_Armah_2015]_
-    - Baseline coverage of MAM needs to be updated, efficacy comes from trial and may be too optimistic
+    - 15 x 0.731 = 0.11
+    - 1 - 0.11 = 0.89
+    - Efficacy: [Ackatia_Armah_2015]_, Coverage: assumption from CIFF/UNICEF
+    - Efficacy comes from trial and may be too optimistic
   * - MAM alternative
-    - 90%
+    - 70%
     - 75% for RUSF
-    - 0.9 x 0.75 = 0.675
-    - 1 - 0.675 = 0.325
+    - 0.9 x 0.75 = 0.53
+    - 1 - 0.53 = 0.47
     - Sphere standards
     - Sphere guideline for efficacy only
```

Comment:
Where does the coverage assumption of 70% come from?

Reply:
This was suggested as a more realistic target by CIFF/UNICEF at our most recent meeting. I have added this as a note to the concept model, thanks.